main.c

```diff
@@ -234,7 +236,11 @@ static void handle_wl_output_mode(void *data, struct wl_output *output,
 }
 
 static void handle_wl_output_done(void *data, struct wl_output *output) {
-	// Who cares
+	struct swaylock_surface *surface = data;
+	if (!surface->created && surface->state->run_display) {
+		create_surface(surface);
+		wl_display_roundtrip(surface->state->display);
```

Comment:
I don't think this is necessary (in fact, I think it's a potential source of bugs), but it was there before so orthogonal to this PR.

Reply:
https://github.com/swaywm/swaylock/pull/336